app/src/main/java/nodomain/freeyourgadget/gadgetbridge/service/devices/miband2/operations/FetchSportsDetailsOperation.java

```diff
@@ -97,7 +97,7 @@ protected void handleActivityFetchFinish(boolean success) {
             try {
                 ActivityTrack track = parser.parse(buffer.toByteArray());
                 ActivityTrackExporter exporter = createExporter();
-                String fileName = FileUtils.makeValidFileName("gadgetbridge-track-" + DateTimeUtils.formatIso8601(summary.getStartTime()));
+                String fileName = FileUtils.makeValidFileName("gadgetbridge-track-" + DateTimeUtils.formatIso8601(summary.getStartTime())+ ".gpx");
```

Comment:
The idea is that we can have different kinds of exporters. The gpx-Exporter may create files with a `.gpx` extension, but other exporters may use a different extension. So we should add a method ActivityTrackExpoter#getExportFileExtension() and use its result here.